### ISSUE TITLE
Provide a prebuilt PDF book in the releases

### ISSUE THREAD

**Issue by cblc** [paroj/gltut#148]
It would be very desirable that you provide a prebuilt PDF version of the book. When you released 0.3.x, you did. But in the latest 0.4.0, you didn't. I belong to the "low on dependencies" trend, so getting the PDF without having to check if I have all the software for building it, would be really nice.

**Comment by gybo03** [comment]
yes this would be more convenient since i just had to change path to my xsl stylesheets location because the CopyWebsite.lua did not detect it correctly
